cms/static/sass/views/_settings.scss

```diff
@@ -978,3 +986,100 @@
     }
   }
 }
+
+// +Settings - Licenses
+// ====================
+.view-settings {
+
+  .license-img {
+    padding: ($baseline/5);
+  }
+
+  .license-types {
+    text-align: center;
+    vertical-align: middle;
+    display: inline-block;
+
+    .license-type {
+      display: inline-block;
+    }
+
+    .action.license-button {
+      @include grey-button;
+      @extend %t-action2;
+      display: inline-block;
+      text-align: center;
+      width: 220px;
+      height: 40px;
+      cursor: pointer;
+
+      &.is-selected {
+        @include blue-button;
+      }
+    }
+
+    .tip {
+      @extend %t-copy-sub2;
+    }
+  }
+  .wrapper-license-options {
+    margin-bottom: ($baseline/2);
+
+    .tip {
+      @extend %t-copy-sub2;
+    }
+    .license-options {
+      padding-bottom: ($baseline/2);
+
+      .license-option {
+        line-height: 1.5;
+        border-bottom: 1px solid $gray-l4;
+        padding: ($baseline/2) 0 ($baseline*0.4);
+
+        &.is-clickable {
+          cursor: pointer;
+        }
+
+        &:last-child {
+          border-bottom: none;
+        }
+
+        input[type=checkbox] {
+          vertical-align: top;
+          width: auto;
+          min-width: auto;
+          height: auto;
+          border: 0;
+          margin: ($baseline*0.15) 15px 0px;
```

Comment:
yup, you could use @include margin(top right bottom left); here as well. In this case 15px is being applied to both left and right, but we could optionally move to RTL syntax if we like.